src/PSADT/PSADT/Module/PSADT.Module.Class.DeploymentSession.cs

```diff
@@ -1076,7 +1076,9 @@ private void CreateDeferHistoryPath()
         /// </summary>
         /// <param name="deferDeadline">The deferral deadline.</param>
         /// <param name="deferTimesRemaining">The deferral times remaining.</param>
-        public void SetDeferHistory(int? deferTimesRemaining, string deferDeadline)
+        /// <param name="deferRunInterval">The interval in minutes before prompting the user again after a deferral.</param>
+        /// <param name="deferRunIntervalLastTime">The timestamp of the last deferRunInterval.</param>
+        public void SetDeferHistory(int? deferTimesRemaining, string deferDeadline, int? deferRunInterval = null, string deferRunIntervalLastTime = null)
```

Comment:
The new string should be nullable based on the setup performed on the PowerShell side.